provider/openstack/provider.go

```diff
@@ -703,17 +703,15 @@ func (e *Environ) PrepareForBootstrap(ctx environs.BootstrapContext, controllerN
 		return err
 	}
 	if !e.supportsNeutron() {
-		logger.Warningf(`Using deprecated OpenStack APIs.
+		logger.Warningf(`Using unsupported OpenStack APIs.
```

Comment:
@mitechie: Thoughts on wording?

Reply:
Let's go ahead and make this an error since it's a bad bad thing that we know won't work.